rfcs/text/0046/rule.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: rule
+  fields:
+    - name: remediation
+      level: extended
+      type: wildcard
```

Comment:
Would remediations usually be human-readable? In that case, using `text` type could be better than wildcard: https://www.elastic.co/guide/en/elasticsearch/reference/current/keyword.html#wildcard-field-type